photon-version.json

```diff
@@ -1,5 +1,4 @@
 {
-  "version": "0.4.0",
```

Comment:
So this is not required anymore? How does this JSON pull in the version from the BOM then?

Reply:
It pulls directly from the BOM rather than from this specific JSON. The values eventually make it into the build scripts, so we're simply removing it from `photon-version.json` which is Packer specific config to our new BOM